packages/locales/src/nl.ts

```diff
@@ -6,8 +6,10 @@ import { str } from '@lit/localize';
 export const templates = {
   s00a84204340a1db4: 'Geselecteerd',
   s078cac19db271b0d: 'Stap omlaag',
+  s08cb5d07b56d33a7: 'wk',
```

Comment:
Maybe we can use somehow locale for it? Right now we have day names translated based on locale, and `wk` week is not translated in all languages, so we have inconsistencies in the calendar header.

Reply:
```
new Intl.DisplayNames("nl", { style: 'short', type: "dateTimeField" }).of('weekOfYear')
``` return `'wk'` 💯